gotham-server/src/server.rs

```diff
@@ -55,3 +63,15 @@ pub fn get_server() -> Rocket {
         )
         .manage(config)
 }
+
+fn get_db() -> db::DB {
+    let is_aws_db = match env::var("DB") {
+        Ok(v) => v == "AWS",
+        _ => false,
+    };
+    if is_aws_db {
+        db::DB::AWS(DynamoDbClient::new(Region::UsWest2))
```

Comment:
Ideally you would want the region to be also a config that is injected. We could use https://docs.rs/config/0.9.2/config/, if you want we can open an issue and do that as a separate step.

Reply:
Done.